scripts/vm/network/security_group.py

```diff
@@ -61,6 +64,15 @@ def execute(cmd):
     logging.debug(cmd)
     return bash("-c", cmd).stdout
 
+def load_kernel_module(module_name):
+    module_proc_file = "/proc/modules"
+    idx = execute("grep -E '^%s' %s | cut -f1 -d' '" % (module_name, module_proc_file)).find(module_name)
+    if idx == -1:
+        logging.debug("Module %s is absent. Load it." % (module_name))
+        execute("modprobe %s" % (module_name))
```

Comment:
Can this throw an error, should we try-catch it?

Reply:
Hi, please, take a look at #2321, I think the solution when module is installed during dpkg install is better than this one and this PR should be declined.